src/test/librbd/operation/test_mock_SnapshotRemoveRequest.cc

```diff
@@ -148,7 +148,11 @@ TEST_F(TestMockOperationSnapshotRemoveRequest, Success) {
 
   C_SaferCond cond_ctx;
   MockSnapshotRemoveRequest *req = new MockSnapshotRemoveRequest(
-    mock_image_ctx, &cond_ctx, "snap1", snap_id);
```

Comment:
You prefer the original version, but wrap the line as it exceeds the line length or not to wrap at all even if it exceeds the line length?